CMakeLists.txt

```diff
@@ -516,9 +572,9 @@ foreach(target devilution ${BIN_TARGET} SourceS)
   target_compile_definitions(${target} PRIVATE ${def_list})
 endforeach(target)
 
-if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND NOT HAIKU)

```

Comment:
We do want to link the C++ stdlib statically in most cases. Perhaps add `AND NOT VITA` instead of commenting this out?

Reply:
Done